src/smefit/analyze/report.py

```diff
@@ -546,27 +543,43 @@ def fisher(
                     fisher_cal.quad_fisher, self.data_info, norm, log
                 )
 
-            # Write down the table in latex
-            free_coeff_config = self.coeff_info.loc[
-                :, fit.coefficients.free_parameters.index
-            ]
             compile_tex(
                 self.report,
-                fisher_cal.write_grouped(
-                    free_coeff_config, self.data_info, summary_only
-                ),
+                fisher_cal.write_grouped(self.coeff_info, self.data_info, summary_only),
                 f"fisher_{fit.name}",
             )
             links_list.append((f"fisher_{fit.name}", f"Table {fit.label}"))
 
             if plot is not None:
                 fit_plot = copy.deepcopy(plot)
+                fit_plot.pop("together")
```

Comment:
This line makes it crash in case together is not specified in the runcard. I think it would be better if it is made sure that if it's not there, it gets set to a default value?